code/game/objects/_items.dm

```diff
@@ -98,6 +98,12 @@
 	//Items are not dense typically
 	can_block_movement = FALSE
 
+	//Tooltip vars
+	///string form of an item's force. Edit this var only to set a custom force string
+	var/force_string
```

Comment:
string form of force? You mean the number that indicates how much damage it does? Why would you cache that

Reply:
> string form of force? You mean the number that indicates how much damage it does? Why would you cache that

Reverted